deployment/roles/openzaak/templates/deployment.yml.j2

```diff
@@ -74,7 +74,7 @@ spec:
           - name: DJANGO_SETTINGS_MODULE
             value: openzaak.conf.docker
           - name: ALLOWED_HOSTS
-            value: "localhost,{{ domain }}"
+            value: "localhost,nginx.openzaak-test,{{ domain }}"
```

Comment:
nasty - inway could be adapted to pass a HTTP_HOST header, or we can set it in nginx config to pass the canonical Host header.

Reply:
> we can set it in nginx config to pass the canonical Host header

This sounds like a reasonable solution.